app/src/main/java/com/example/myweatherapp/data/entity/Data.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.myweatherapp.data.entity
+
+data class Data(
```

Comment:
please give the object a more significant name

Reply:
Done, thx